Makefile

```diff
@@ -385,9 +385,14 @@ $(BUILD_DIR)/presences.ttl: $(BUILD_DIR)/phenex-data+tbox.ttl $(SPARQL)/presence
 	$(ARQ) --data=$< --query=$(SPARQL)/presences.sparql > $@
 
 # Generate taxon-profiles.ttl
-$(BUILD_DIR)/taxon-profiles.ttl: $(BUILD_DIR)/phenex-data+tbox.ttl $(SPARQL)/taxonProfiles.sparql
+$(BUILD_DIR)/taxon-profiles.ttl: $(BUILD_DIR)/evolutionary-profiles.ttl $(SPARQL)/taxonProfiles.sparql
```

Comment:
@Shalsh23 do you think this step is needed at all? Or could `evolutionary-profiles.ttl` just be in place of wherever `taxon-profiles.ttl` is referenced?